spec/swagger_schemas/attributes/id.rb

```diff
@@ -1,5 +1,5 @@
 ID_ATTRIBUTE = {
   description: "The unique ID of the resource.",
-  type: :uuid,
+  type: :string,
```

Comment:
Why is this changing, out of interest?

Reply:
I used https://editor.swagger.io/ to validate the swagger.yaml file and there was an error on the type: :uuid which is not a valid type according to https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.1.md#data-types so had to add `type: :string, format: :uuid` instead.